api/app.js

```diff
@@ -68,6 +69,7 @@ app.use(['/api/recipient', '/api/recipients'], recipientRouter);
 app.use(['/api/household', '/api/households'], householdRouter);
 app.use(['/api/metric', '/api/metrics'], metricsRouter);
 app.use(['/api/veteran', '/api/veterans'], veteransRouter);
+app.use(['/api/location', '/api/locations'], locationRouter);
```

Comment:
Great job here with creating the endpoints and following the pre-existing convention!